### PR TITLE
update info about changing order

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4480

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-app-setup-policies.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ With the Teams mobile clients, users will see core Teams apps such as Activity,
 
 #### Can users change the order of apps pinned through a policy?
 
-Currently, users can change the order of their pinned apps on Teams mobile clients but not on the Teams desktop or web clients.
+Users can change the order of their pinned apps on Teams mobile clients, Teams desktop and web clients, if the option "Allow user pinning" is set to On. More details [here](https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies)
```
Comment (JohanFreelancer9) [inline Teams/teams-app-setup-policies.md:180]:
```suggestion
Users can change the order of their pinned apps on Teams mobile clients and Teams desktop and web clients if the option "Allow user pinning" is set to On. More details [here](https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies)
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@get2chhavi please review this PR and let me know whether it's OK to merge.

**Pull request merged**